ionic-angular-template/src/app/home/home.module.ts

```diff
@@ -16,7 +24,14 @@ import { HomePage } from './home.page';
         path: '',
         component: HomePage
       }
-    ])
+    ]),
+    TranslateModule.forRoot({
```

Comment:
Can we move this to the app.module.ts

Reply:
I got this error "The pipe 'translate' could not be found". I checked it and found out that my home page is lazily loaded and therefore I have to add these configs to home.module as well.

Reply:
Can I do a shared module instead of adding these configs every time I need it ?

Reply:
I think that the initialization should be in the app.module.ts and then you should import in every module.ts the `TranslateModule`

For example: `import { TranslateModule } from '@ngx-translate/core';`

Then: 
`@NgModule({
  imports: [
    CommonModule,
    FormsModule,
    ReactiveFormsModule,
    TranslateModule,
    IonicModule,
    SharedModule,
    RouterModule.forChild(routes)
  ],
  declarations: [ForgotPasswordPage, EnterEmailComponent]
})`